linera-views/tests/operations_tests.rs

```diff
@@ -100,20 +84,20 @@ async fn test_readings_vec<OP: KeyValueStoreClient + Sync>(
     for _ in 0..10 {
         let mut keys = Vec::new();
         let mut values = Vec::new();
-        for key_value in &key_value_vec {
+        for (key, value) in &key_value_vec {
             if rng.gen() {
                 // Put a key that is already present
-                keys.push(key_value.0.clone());
-                values.push(Some(key_value.1.clone()));
+                keys.push(key.clone());
+                values.push(Some(value.clone()));
             } else {
                 // Put a missing key
-                let len = key_value.0.len();
+                let len = key.len();
                 let pos = rng.gen_range(0..len);
-                let byte = *key_value.0.get(pos).unwrap();
+                let byte = *key.get(pos).unwrap();
                 let new_byte: u8 = if byte < 255 { byte + 1 } else { byte - 1 };
-                let mut new_key = key_value.0.clone();
+                let mut new_key = key.clone();
                 *new_key.get_mut(pos).unwrap() = new_byte;
-                if !set_keys.contains(&new_key) {
+                if !set_keys.contains(&*new_key) {
```

Comment:
I am not sure that `&*new_key` is such a simplification.

Reply:
It's necessary now because `set_keys` doesn't contain `Vec`s: It contains `&[u8]`s.
So `contains` won't accept a `&Vec<u8>` as an argument anymore.